repairnator/src/main/java/fr/inria/spirals/repairnator/process/step/TestPreviousBuild.java

```diff
@@ -0,0 +1,47 @@
+package fr.inria.spirals.repairnator.process.step;
+
+import java.io.File;
+import java.io.IOException;
+
+import org.eclipse.jgit.api.Git;
+import org.eclipse.jgit.api.errors.GitAPIException;
+
+import fr.inria.spirals.jtravis.entities.Build;
+import fr.inria.spirals.repairnator.process.inspectors.ProjectInspector;
+import fr.inria.spirals.repairnator.process.inspectors.ProjectInspector4Bears;
+
+/**
+ * Created by fermadeiral.
+ */
+public class TestPreviousBuild extends CloneRepository {
+
```

Comment:
In fact you don't test previous build there, but you're checking out the repository to put it in the same test, so the name is wrong.